### ISSUE TITLE
🐛 BUG: ProxyWorker request logs

### ISSUE THREAD

**Issue by Cherry** [cloudflare/workers-sdk#4168]
### Which Cloudflare product(s) does this pertain to?

Workers Runtime

### What version(s) of the tool(s) are you using?

3.13.0

### What version of Node are you using?

18.18.0

### What operating system are you using?

Windows 10

### Describe the Bug

With wrangler 3.13.0, I'm now seeing the following in my logs regularly when developing:
```
[wrangler:inf] GET /cdn-cgi/ProxyWorker/pause 204 No Content (0ms)
[wrangler:inf] GET /cdn-cgi/ProxyWorker/pause 204 No Content (12ms)
[wrangler:inf] GET /cdn-cgi/ProxyWorker/pause 204 No Content (1ms)
[wrangler:inf] GET /cdn-cgi/ProxyWorker/play 204 No Content (1ms)
```

Is this intended? Should these be debug logs instead? These logs do not appear in 3.12.0 or prior.

**Comment by admah** [comment]
@Cherry we've identified some issues with the latest version of Wrangler (3.13.0) and are working on fixes. In the meantime, we recommend staying on Wrangler 3.12.0 (npm install wrangler@latest should point to 3.12.0).

**Comment by admah** [comment]
we have published a new version of Wrangler (3.13.1) that reverts the change that was causing issues. You should be able to update and not see these logs.

**Comment by RamIdeas** [comment]
This has been fixed in #4175. Since this bug is only present in that branch and not main, I will close this issue now. Thanks for reporting!
